Frameworks/Foundation/NSPathUtilities.mm

```diff
@@ -39,15 +39,15 @@
 // Helper that gets the path for a folder dirName under the current app's AppData/Local... directory,
 // creating the folder if necessary
 NSString* _getCreateAppDataLocalDir(const char* dirName) {
-    auto ret = [NSString stringWithFormat:@"%S/%hs", IwGetWritableFolder(), dirName];
+    auto ret = [NSString stringWithFormat:@"%ls/%hs", IwGetWritableFolder(), dirName];
     _mkdir([ret cStringUsingEncoding:NSUTF8StringEncoding]);
     return ret;
 }
 
 // Override for when a higher-level directory needs to be created first (eg: Foo1/Foo2/)
 NSString* _getCreateAppDataLocalDir(const char* dirName1, const char* dirName2) {
     _getCreateAppDataLocalDir(dirName1);
-    auto ret = [NSString stringWithFormat:@"%S/%hs/%hs", IwGetWritableFolder(), dirName1, dirName2];
```

Comment:
>%S/%hs [](start = 44, length = 6)

Is there an objection to %S? That's standard.

And no objection to %hs, which is a Microsoft extension?